diffsims/tests/test_generators/test_rotation_list_generator.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The diffsims developers
+#
+# This file is part of diffsims.
+#
+# diffsims is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# diffsims is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with diffsims.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+import numpy as np
+from diffsims.generators.rotation_list_generators import get_local_grid, get_grid_around_beam_direction,get_fundemental_zone_grid
+from diffsims.utils.rotation_conversion_utils import Euler
+
+
+@pytest.mark.parametrize("center",[(0,0,0),(0,10,0)])
+def test_get_local_grid(center):
+    grid = get_local_grid(center,10,2)
+    assert isinstance(grid,list)
+    assert isinstance(grid[0],tuple)
```

Comment:
In this test and the one below - would it be possible to assert a couple of actual values rather than type only? I know we've done this elsewhere, but I think we should avoid it really.

Reply:
Fair play, this caught a bug.